neqo-bin/src/lib.rs

```diff
@@ -111,6 +111,10 @@ pub struct QuicParameters {
     /// The idle timeout for connections, in seconds.
     pub idle_timeout: u64,
 
+    #[arg(long = "init_rtt", default_value = "100")]
```

Comment:
```suggestion
    #[arg(long = "init_rtt", default_value = DEFAULT_INITIAL_RTT)]
```

Reply:
Can the macro take a `Duration` like that?  I thought that it had to be a string.  The other way to spell this is to use `Option<u64>` and use `map_or` to get the default.

Reply:
Yes, not sure that it can take a `Duration`. Was trying to suggest to avoid violating POLA and somehow use the same default here that the code is using.

Reply:
I'd use Option and move the application of the default to the getter.  That works, even if it means that the `--help` doesn't report the default.